### ISSUE TITLE
AMASS for training

### ISSUE THREAD

**Issue by babyfisher** [akashsengupta1997/STRAPS-3DHumanShapePose#9]
Hi,
Thank you for sharing your great work!
I noticed that you provided preprocessed AMASS to replace H3.6M for training. Did you use a subset of AMASS to get a similar data scale as H3.6M? What datasets in AMASS did you use? Do you use whole sequences or do you skip frames?

**Comment by akashsengupta1997** [comment]
Hey sorry for the very late reply!

I believe I used the entirety of AMASS (or the version of it that was current at the time this repo was released). This did result in ~300k training poses, compared to around ~60k for H3.6M. However, I didn't seem to get much improvement in pose evaluation metrics despite using the larger training dataset, which is kind of curious. I haven't investigated why, but it could be related diversity of poses in AMASS vs H3.6M?

I sampled frames such that I obtained 2 training poses per second of motion capture.
